src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -431,8 +427,7 @@ public void periodic() {
    * move as soon as the lift is clear of that zone
    */
   public boolean clearOfIntakeZone() {
-    if ((elevatorLeftEncoder.getPosition() > Cal.Lift.ELEVATOR_INTAKE_ZONE_THRESHOLD_INCHES)
-        || (armEncoder.getPosition() > Cal.Lift.ARM_INTAKE_ZONE_THRESHOLD_DEGREES)) {
+    if (armEncoder.getPosition() > Cal.Lift.ARM_INTAKE_ZONE_THRESHOLD_DEGREES) {
```

Comment:
Just return this value instead of having a conditional.

Reply:
Good call